integration-tests/docker-compose-configs/docker-compose.base.yaml

```diff
@@ -8,6 +8,12 @@ services:
     command: "--sep-server"
     environment:
       - STELLAR_ANCHOR_CONFIG=file:/config/anchor-platform-config.yaml
+      - SECRET_SEP10_SIGNING_SEED=SAX3AH622R2XT6DXWWSRIDCMMUCCMATBZ5U6XKJWDO7M2EJUBFC3AW5X
```

Comment:
I think Thibault is going to pick on this pattern. Can we submit an issue for the secrets used in tests?